src/components/form/types.js

```diff
@@ -1,8 +1,9 @@
 export const COUNTRIES = {
   UK: 0,
-  INDIA: 1,
-  SOUTH_AFRICA: 2,
+  IN: 1,
+  ZA: 2,
   OTHER: 3,
+  US: 4,
 }
```

Comment:
Are we storing the country in attributes? If so, do we want to ensure we're storing a format that is human readable?

Reply:
not yet, maybe store the country code like US, UK, IN, ZA

Reply:
Lets goo with  US, UK, IN, ZA for now, I'll communicate Sonia they should look for these codes, unless they need different.